packages/test/local-server-stress-tests/src/localServerStressHarness.ts

```diff
@@ -1431,13 +1428,24 @@ function mixinRestartClientFromPendingState<TOperation extends BaseOperation>(
 			assert(sourceClientIndex !== -1, `Client ${op.sourceClientTag} not found`);
 			const sourceClient = state.clients[sourceClientIndex];
 
+			// AB#45904: Clarify restart-from-pending behavior in staging mode
```

Comment:
Up to you but I'd call this fixed.  The new item to maybe open would be "Add support for serializing pending state while in staging mode".  In fact, how about we add a new UsageError at the top of ContainerRuntime.getPendingLocalState if we're in Staging Mode, to make it official.  Then work with your team to figure out how to track that future feature (it's more than a task)